### PR TITLE
General housekeeping for 1.0 release

### PR DESCRIPTION
**Pull request by yakovkhalinsky**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+### v1.0.0 (March 15, 2017) - The large files release
+
+Features
+- Added large file API support [link](https://github.com/yakovkhalinsky/backblaze-b2/pull/22)
+
+Thanks to the contributors for this HUGE release
+- [crazyscience](https://github.com/crazyscience)
```
Comment (crazyscience) [inline CHANGELOG.md]: @yakovkhalinsky @IvanKalinin did most of the work :P

Comment (yakovkhalinsky) [inline CHANGELOG.md]: I'll credit him then (commit coming), I'm pretty stoked with this release guys 👍

Comment (crazyscience) [inline CHANGELOG.md]: Ah thanks. We're pretty pleased with it as well! It was definitely an interesting project.

**Comment by yakovkhalinsky** [comment]
Alright @crazyscience @IvanKalinin once the build passes I'll publish this to npm 👍

**Comment by yakovkhalinsky** [comment]
@crazyscience @IvanKalinin upped to npm, thanks again 👍

**Pull request merged**